src/components/todoList/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+export const TodoList = ({ todo, deleteTodo, updateTodo }) => {
+  const [checked, setChecked] = useState(true);
+  const [newName, setNewName] = useState(todo.name);
+  const [editing, setEditing] = useState(false);
+
+  const toggleCheckBox = () => {
+    setChecked((prevState) => !prevState);
+    todo.completed = checked;
+  };
+
+  const editTodo = (e) => {
+    if (e.key === "Enter") {
+      updateTodo(todo.id, newName);
```

Comment:
you could've used e.target.value to retrieve newName, no need for extra state here